dispersydatabase.py

```diff
@@ -84,255 +84,13 @@ def check_database(self, database_version):
             self.commit()
 
         else:
-            # upgrade an older version
+            # Check if the version is not higher than
+            if database_version > LATEST_VERSION:
+                raise RuntimeError(u"Your database version exceeds the latest version.")
```

Comment:
@devos50 the solution to this would be to have versioned statedirs. There's an issue talking about that (should be trivial to implement).